message_ix_models/model/water/data/water_for_ppl.py

```diff
@@ -85,26 +96,36 @@ def shares(
     corresponding cooling fraction
     """
     for col in search_cols_cooling_fraction:
-        # MAPPING ISOCODE to region name, assume one country only
         col2 = context.map_ISO_c[col] if context.type_reg == "country" else col
+
+        # Filter the cooling fraction
         cooling_fraction = hold_df[
             (hold_df["node_loc"] == col2)
             & (hold_df["technology_name"] == x["technology"])
         ]["cooling_fraction"]
-        x[col] = x[col] * cooling_fraction
 
-    results: list[Any] = []
+        # Log unmatched rows
+        if cooling_fraction.empty:
+            log.warning(
```

Comment:
Should this be a `warning`? This looks like it's going to clutter the output potentially, so you could also think of making this `info` or `debug(ing)`.

Reply:
changed to info. thanks!